src/core/connection.c

```diff
@@ -7287,6 +7299,15 @@ QuicConnProcessApiOperation(
         }
         break;
 
+    case QUIC_API_TYPE_CONN_COMPLETE_RESUMPTION_TICKET_VALIDATION:
+        CXPLAT_DBG_ASSERT(QuicConnIsServer(Connection));
+
+        Status =
+            QuicCryptoCustomTicketValidationComplete(
```

Comment:
This is an async operation from the app. They don't wait on it, so they can't get any errors back. Therefore, there's no reason for this function to return an error code at all.